frontend/express/app.js

```diff
@@ -916,6 +931,8 @@ Promise.all([plugins.dbConnection(countlyConfig), plugins.dbConnection("countly_
                 path: countlyConfig.path || "",
                 cdn: countlyConfig.cdn || "",
                 message: req.flash("message"),
+                notify,
+                license,
```

Comment:
@kanwarujjaval license is an error message for frontend. It's empty if we're all good, `license.err.restricted` if instance is locked (only license manager for global admin case) and `license.err.locked` if instance is restricted (prevent login for non global admins case).